### PR TITLE
Modified ppc_loo_pit_overlay warning message for discrete values

### PR DESCRIPTION
**Pull request by ecoronado92**
@avehtari Just modified the warning message to reflect your suggestion

PS. Also, congrats to you and @jgabry for the recent arXiv paper on Bayesian workflows!! I liked it a lot!

### CODE REVIEW

**Comment by avehtari** [comment]
Why the note is inside
```
 if (boundary_correction) 
```
The same note holds without boundary correction.

File: R/ppc-loo.R [hunk 1]
```diff
@@ -167,7 +167,8 @@ ppc_loo_pit_overlay <- function(y,
   }
 
   if (boundary_correction) {
```
Comment (avehtari) [inline R/ppc-loo.R:172]: Maybe move the NOTE message out of this if statement? It hold also for the old kernel density estimate.

**Comment by ecoronado92** [comment]
@avehtari Message will now display for both bc and non-bc versions

**Comment by avehtari** [comment]
Great, thanks for fixing this!

**Review by avehtari — APPROVED** [review]

**Comment by jgabry** [comment]
Thanks!

**Pull request merged**